sdk/notificationhubs/notification-hubs/vitest.int.config.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { defineConfig, mergeConfig } from "vitest/config";
+import viteConfig from "../../../vitest.shared.config.ts";
```

Comment:
put this shared config in dev-tool perhaps and import from there instead of global config files?

Reply:
I think that in the root is fine for now because we have our other configs there and so far dev-tool isn't involved here.